src/safeStringify.ts

```diff
@@ -1,9 +1,9 @@
-import fastStringify from '@stoplight/fast-safe-stringify';
+import fastStringify from 'safe-stable-stringify';
```

Comment:
Could we add a test case for the issues described in the ticket?
Might prevent us from a potential regression in the future.

Reply:
It's going to be pretty difficult - it's a 20k line specification, that turns into 286k lines when dereferenced. Some circular edge case in there is not being handled correctly by fast-safe-stringify :/.

Reply:
Gotcha, gotcha.
Perhaps leave a comment there explaining the reasoning behind choosing the following package in case somebody decides to switch it for some reason?
Just saying, nothing compulsory.